modules/monitoring-moving-monitoring-components-to-different-nodes.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// monitoring/configuring-the-monitoring-stack.adoc
+
+[id='moving-monitoring-components-to-different-nodes-{context}']
+= Moving monitoring components to different nodes
+
+You can move the monitoring stack components to specific nodes.
+
+.Prerequisites
+
+* Make sure you have the `cluster-monitoring-config` ConfigMap object with the `data/config.yaml` section.
+
+.Procedure
+
+. Start editing the `cluster-monitoring-config` ConfigMap:
++
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Specify the `nodeSelector` constraint for the component under `data/config.yaml`:
++
+[source,yaml,subs=quotes]
+----
+data:
+ config.yaml: |
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+   name: cluster-monitoring-config
+   namespace: openshift-monitoring
+  data:
+   config.yaml: |
+     *_component_*:
+       nodeSelector:
+         *_node key_*: *_node value_*
+         *_node key_*: *_node value_*
+         *_..._*
+----
++
+Substitute `<component>` accordingly and substitute `<node key>: <node value>` with the map of key-value pairs that specifies the destination node. Often, only a single key-value pair is used.
++
+The component can only run on a node that has each of the specified key-value pairs as labels. The node can have additional labels as well.
++
+For example, to move the Prometheus and Alertmanager components to the node labeled `region: us`, use:
++
+[source,yaml,subs=quotes]
+----
+data:
+ config.yaml: |
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: cluster-monitoring-config
+    namespace: openshift-monitoring
+  data:
+    config.yaml: |+
+      *alertmanagerMain:
+        nodeSelector:
+          region: us
+      prometheusK8s:
+        nodeSelector:
+          region: us*
+----
+
+. Apply the new configuration:
```

Comment:
No need for this step, and the command will return error.
Just save the configmap is OK, the configuration will be applied.

#oc -n openshift-monitoring apply configmap cluster-monitoring-config
Error: required flag(s) "filename" not set